### DISCUSSION TITLE
📅  Fecha del evento!

### DISCUSSION THREAD

**Discussion by nucliweb** [WEDeveloperConf/organizacion#1] [Announcements]
## 📅  Fecha del evento!

Discusión para tratar la fecha para el evento

**Comment by sanguino** [comment]
Primavera?

**Comment by pablolopezmestre** [comment]
¡Hola!

¿Nos juntamos y empezamos a mover esto?

¡Saludos!

**Comment by sanguino** [comment]
Nos hemos quedado un poco fríos con esto. Retomamos?

**Reply by reiaguilera** [reply]
Yo quiero retomar este tema
